app/src/components/datasets/DatasetContentTabs/General/DatasetEntriesTable/DatasetEntryEditorDrawer.tsx

```diff
@@ -106,67 +111,120 @@ export default function DatasetEntryEditorDrawer({
-          </HStack>
+          <VStack w="full" alignItems="flex-start">
+            <HStack w="full" justifyContent="space-between" pr={12}>
+              <Heading size="md">Dataset Entry</Heading>
+              {datasetEntry && (
+                <EntrySplitDropdown split={datasetEntry.split} onChange={onUpdateSplit} />
+              )}
+            </HStack>
+          </VStack>
         </DrawerHeader>
         <DrawerBody h="full" pb={4} bgColor="orange.50">
-          <VStack h="full" justifyContent="space-between">
-            <VStack w="full" spacing={12} py={4}>
-              <VStack w="full" alignItems="flex-start">
-                <Text fontWeight="bold">Input</Text>
-                {inputMessagesToSave.map((message, i) => {
-                  return (
-                    <>
-                      <Divider key={`divider-${i}`} my={4} />
-                      <EditableMessage
-                        key={i}
-                        message={message}
-                        onEdit={(message) => {
-                          const newInputMessages = [...inputMessagesToSave];
-                          newInputMessages[i] = message;
-                          setInputMessagesToSave(newInputMessages);
-                        }}
-                        onDelete={() => {
-                          const newInputMessages = [...inputMessagesToSave];
-                          newInputMessages.splice(i, 1);
-                          setInputMessagesToSave(newInputMessages);
-                        }}
-                        ruleMatches={datasetEntry?.matchedRules}
-                      />
-                    </>
-                  );
-                })}
-                <Divider my={4} />
-                <Button
-                  w="full"
-                  onClick={() =>
-                    setInputMessagesToSave([...inputMessagesToSave, { role: "user", content: "" }])
-                  }
-                  variant="outline"
-                  color="gray.500"
-                  _hover={{ bgColor: "orange.100" }}
-                >
-                  <HStack spacing={0}>
-                    <Text>Add Message</Text>
-                    <Icon as={BsPlus} boxSize={6} />
+          <VStack w="full" spacing={12} pb={4}>
+            {datasetEntry && (
+              <VStack
+                w="full"
+                alignItems="flex-start"
+                fontSize="xs"
+                bgColor="gray.50"
+                borderRadius={8}
+                borderWidth={1}
+                borderColor="gray.200"
+                p={4}
+              >
+                {datasetEntry?.importId && (
+                  <>
+                    <Text>
+                      <Text as="span" fontWeight="bold">
+                        Import ID:
+                      </Text>{" "}
+                      {datasetEntry.importId}
+                    </Text>
+                  </>
+                )}
+                <Text>
```

Comment:
We might be able to both fit the visual styling of the drawer better and take up less space by removing the outline/different-colored background but not blocking, I can play with that later.